CHANGELOG.md

```diff
@@ -1,3 +1,449 @@
+## v0.3.0 - 2020-07-30
```

Comment:
31 :)

Reply:
Probably on 08-03 as @ipochi is not around today :smile:

Reply:
Today :)

Reply:
done. will need you approvals again.